### ISSUE TITLE
No menu displayed in Ubuntu 16.04

### ISSUE THREAD

**Issue by asitemade4u** [QTodoTxt/QTodoTxt2#96]
Hi, and thank you for this excellent software which I have been using since now in version 1.x
When switching to version 2, I lost the main menu, which is annoying as there seems to be a lot of new functions in the Preferences dialog -- which I cannot access now.
Please help.
Best,
Stephen
![selection_001](https://user-images.githubusercontent.com/11460106/38064672-476260f2-32cd-11e8-80a1-a94a421657e3.png)

**Comment by asitemade4u** [comment]
After looking around for the configuration file, I have found QTodoTxt2.conf
Tried to add the show_menu=true instruction, but it did not trigger the menu to be displayed.

**Comment by oroulet** [comment]
Older version of Ubuntu use menus on top of screen

**Comment by asitemade4u** [comment]
Yes, you are right. However, whatever the way menus are displayed (integrated in the window or not), they are not displayed in QToDoTxt2.

**Comment by oroulet** [comment]
Then it is probably a big/limitation of qml on that Linux version and we cannot do anything at qtodotxt2 level

**Comment by asitemade4u** [comment]
This is a LTS version.

Stephen Sinclair
<http://asitemade4u.com/?utm_source=stephen_email&amp;utm_medium=email&amp;utm_content=logolink&amp;utm_campaign=current_stephens_email>
email: stephen@aSiteMade4U.com <mailto:stephen@aSiteMade4U.com>
web: aSiteMade4U.com 
<http://asitemade4u.com/?utm_source=stephen_email&amp;utm_medium=email&amp;utm_content=logolink&amp;utm_campaign=current_stephens_email>
cell: (914) 334-2929

**Comment by licquia** [comment]
Have you tried moving your mouse over the bar at the top of the screen?  In Ubuntu 16.04, they often hide the top menu bar until the mouse hovers over it.

If that doesn't help, this might be useful information:

https://askubuntu.com/questions/785238/removing-global-menu-bar-ubuntu-16-04

**Comment by HaaiHenkie** [comment]
Hello, I have the same problem in ubuntu 16.04 LTS: menu is nowhere to be found.

**Comment by HaaiHenkie** [comment]
Hello,

I did some further research and found [this issue](https://github.com/keepassxreboot/keepassxc/issues/271). Right at the end it says that it is a Qt bug that is fixed in Qt 5.9.0. Ubuntu 16.04 is using Qt version 5.5.1. So I upgraded with:

`pip3 install --user pyqt5` 

That brings the version to 5.11.2 which did not work at all, probably because it is not compatible with earlier versions. Then I tried:

`pip3 install --user pyqt5==5.10.1`

Now the menu appeared but filtering in QTodoTxt2 was not working. If somebody knows a way to properly upgrade PyQT5 under Ubuntu, it might work.

If you start qtodotxt.pyw from a terminal you see the following error:

`appmenu-qt: handleReparent 138 The given QWindow has no QMenuBar assigned`

I googled and found a working solution: disable appmenu support on a per application basis by setting the UBUNTU_MENUPROXY variable to null. You can do this by giving the following command in a terminal or by launching it with the Alt+F2 shortcut:

`env UBUNTU_MENUPROXY= /your-path-to/QTodoTxt2/bin/qtodotxt.pyw`

Only if I do this I see another error:

`file:///usr/lib/x86_64-linux-gnu/qt5/qml/QtQuick/Controls/Private/MenuContentItem.qml:178: ReferenceError: Acccessible is not defined`

I did not yet notice any impact of this error.

Cheers,

Henk

**Comment by HaaiHenkie** [comment]
The last error is a typo (Acccessible with 3 c's) in that file. I have corrected it myself. Also, has nothing to do with QTodoTxt2
